src/forms/FormFieldDropdownSingleSelection/FormFieldDropdownSingleSelection.styled.tsx

```diff
@@ -61,7 +61,7 @@ export const SingleDropdownWrapper = styled.div`
   padding: 0;
 
   .MuiFormControl-fullWidth {
-    height: 51px;
+    height: 43px;

```

Comment:
The FormFieldDropdownSingleSelect and the PhoneFields share the same height, let's move it to the theme file.